natgeo/mortar
# [SYTO-820] update gallery promo

https://jira.natgeo.com/browse/SYTO-820

This corrects some assorted issues with the gallery promo

to test:

build, use promo configurator to confirm gallery image sizing works properly, as well as the new "photo count" field
